### PR TITLE
1991-removing-error-displayed-in-NDA-template

### PR DESCRIPTION
**Pull request by chetanbhatt10**
Targets #1991

### Description
While opening NDA template we fixed error 

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/settings/agreement/nda/index.blade.php [hunk 1]
```diff
@@ -17,7 +17,20 @@
...
+                @php
+                    $mailSubject = $cfmArray['subject'];
+                    $mailBody = $cfmArray['body'];
+                @endphp
+            @else
+                @php
+                    $mailSubject = '';
+                    $mailBody = '';
+                @endphp
+            @endif
```
Comment (GauravGusain98) [inline resources/views/settings/agreement/nda/index.blade.php:33]: What is this logic doing? Can you explain?

**Review by GauravGusain98 — COMMENTED** [review]
Please check the comment

File: resources/views/settings/agreement/nda/index.blade.php (hunk 1 above)
Comment (chetanbhatt10) [reply resources/views/settings/agreement/nda/index.blade.php:33]: This is Removing  error message  and helps to access body and subject from array whenever needed

Comment (chetanbhatt10) [reply resources/views/settings/agreement/nda/index.blade.php:33]: To check if there is any empty subject and body in any section if it is empty then don't do anything

File: resources/views/settings/agreement/nda/index.blade.php [hunk 2]
```diff
@@ -34,15 +47,15 @@
                                 <div class="col-md-12">
                                     <div class="form-group">
                                         <label for="round_mail_subject">Subject</label>
-                                    <input type="text" name="round_mail_subject" class="form-control" value="{{ $round->{$mailTemplate}['subject'] }}">
+                                    <input type="text" name="round_mail_subject" class="form-control" value="{{ $mailSubject }}">
```
Comment (GauravGusain98) [inline resources/views/settings/agreement/nda/index.blade.php:50]: You can use it like this as well. 
```suggestion
                                    <input type="text" name="round_mail_subject" class="form-control" value="{{ $round->{$mailTemplate}['subject'] ?? '' }}">
```

Just try it once.

File: resources/views/settings/agreement/nda/index.blade.php [hunk 3]
```diff
@@ -34,15 +47,15 @@
...
                                     </div>
                                 </div>
                             </div>
                             <div class="form-row">
                                 <div class="col-md-12">
                                     <div class="form-group">
                                         <label for="round_mail_body">Mail body:</label>
-                                        <textarea name="round_mail_body" rows="10" class="richeditor form-control" placeholder="Body">{{ $round->{$mailTemplate}['body'] }}</textarea>
+                                        <textarea name="round_mail_body" rows="10" class="richeditor form-control" placeholder="Body">{{ $mailBody }}</textarea>
```
Comment (GauravGusain98) [inline resources/views/settings/agreement/nda/index.blade.php:58]:
```suggestion
                                        <textarea name="round_mail_body" rows="10" class="richeditor form-control" placeholder="Body">{{ $round->{$mailTemplate}['body'] ?? '' }}</textarea>
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment.

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Review by GauravGusain98 — COMMENTED** [review]
Looks fine

**Review by GauravGusain98 — APPROVED** [review]

**Pull request merged**